### PR TITLE
Logger added in ReceiveFrames and ReceiveFramesTest classes

### PR DESCRIPTION
**Pull request by DirvaNIcolae**

### CODE REVIEW

File: src/mcu/src/ReceiveFrames.cpp [hunk 1]
```diff
@@ -156,16 +156,19 @@ void ReceiveFrames::processQueue()
...
+    LOG_INFO(MCULogger.GET_LOGGER(),"Processing CAN frame from queue:");
+    LOG_INFO(MCULogger.GET_LOGGER(), fmt::format("CAN ID: 0x{:x}", frame.can_id));
+    LOG_INFO(MCULogger.GET_LOGGER(), fmt::format("Data Length: {}", int(frame.can_dlc)));
+    oss << "Data: ";
+    for (uint8_t itr = 0; itr < frame.can_dlc; ++itr)
+    {
+        oss << fmt::format("{} ", int(frame.data[itr]));
+    }
+
+    // Output the entire message using LOG_INFO
```
Comment (Mihnea9901) [inline src/mcu/src/ReceiveFrames.cpp:170]: comments should be of type /* */

**Comment by Mihnea9901** [comment]
These tests are failing: TestProcessQueue_ForMCU, TestProcessQueue_ForECUUp,  TestProcessQueue_ForAPI, TestProcessQueue_TestFrame, TestPrintFrames. 
I tried to catch the output and it is either null or "\n", so something is not printing correctly..

File: src/mcu/src/ReceiveFrames.cpp (hunk 1 above)
Comment (DirvaNIcolae) [reply src/mcu/src/ReceiveFrames.cpp]: Done.

File: src/mcu/src/ReceiveFrames.cpp [hunk 2]
```diff
@@ -26,7 +26,7 @@ bool ReceiveFrames::receiveFramesFromCANBus()
         int nbytes = read(socket_canbus, &frame, sizeof(frame));
         if (nbytes < 0)
         {
-            std::cerr << "Read Error" << std::endl;
+            LOG_INFO(MCULogger.GET_LOGGER(),"Read Error");
```
[the remark this answers is not in the document]
Comment (DirvaNIcolae) [reply src/mcu/src/ReceiveFrames.cpp]: Done.

**Review by GeorgeCimpoeru — APPROVED** [review]

**Review by Mihnea9901 — APPROVED** [review]

**Pull request merged**